packages/form-layout/src/vaadin-form-item-mixin.js

```diff
@@ -42,6 +42,16 @@ export const FormItemMixin = (superClass) =>
       this.__fieldNode = null;
     }
 
+    connectedCallback() {
+      super.connectedCallback();
+      const spacing = getComputedStyle(this).getPropertyValue('--vaadin-form-item-row-spacing');
```

Comment:
It would be better to show this warning only once per page. Currently, it's displayed for each field.

Reply:
Done.